docs/examples/menu/left-and-right.vue

```diff
@@ -40,7 +39,7 @@ const handleSelect = (key: string, keyPath: string[]) => {
 </script>
 
 <style>
-.flex-grow {
-  flex-grow: 1;
+.el-menu--horizontal > .el-menu-item.img-logo {
```

Comment:
No need to add additional classes.
```suggestion
.el-menu--horizontal > .el-menu-item:nth-child(1) {
```

Reply:
Yep, I did think about this too.